fix: correct link to terraform-website, remove link for tfe (#127)
hashicorp/terraform-docs-common: README.md

## Redirects

> For now, redirects will need to be handled here: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js

Reviewer: ```suggestion
Handle all redirects in this file in the `terraform-website` repository: https://github.com/hashicorp/terraform-website/blob/master/redirects.next.js
```

Suggest changing "For now, redirects will need to be handled here:" to "Handle all redirects in this file in the `terraform-website` repository:" to avoid future tense and passive voice style issues.

Author: Fine with me!
(I did not change that line, it was just marked as such because the Github UI seemed to have removed or added a newline at the end of the file 😄)
